README.md

```diff
@@ -105,7 +105,11 @@ Download the `.dfont` and install it with `Font Book .app`. Both the bitmap
 Grab `CozetteVector.ttf`. If you want to get the bitmap versions to work,
 [follow the instructions from here].
 
-### BSD / Solaris / Haiku /Other
+### FreeBSD
+
+* [ports](https://cgit.freebsd.org/ports/tree/x11-fonts/cozette): `cd /usr/ports/x11-fonts/cozette; make install clean`
+
+### BSD / Solaris / Haiku / Other
```

Comment:
Should probably read "Other BSD" now, yeah.

Reply:
done!